.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: npm build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request: {}
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      - name: Publish to npm
+        run: npm publish --tag next
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.npm_token}}
```

Comment:
@GreyCat Yeah, it obviously can't work, because your secret is called [`NODE_AUTH_TOKEN`](https://github.com/kaitai-io/kaitai_struct_javascript_runtime/settings/secrets), not `npm_token`.

Reply:
Got it, thanks! Fixed, let's try it again.